viewIssue.php

```diff
@@ -0,0 +1,27 @@
+<?php
+include('config/init.php');
+
+$result= getAllIssues();
```

Comment:
I don't think you're using this anywhere. Doesn't matter right now, but this will take longer and longer the more issues you have.